src/content/smart-contracts/index.md

```diff
@@ -5,21 +5,15 @@ lang: en
 sidebar: true
 ---
 
-# Introduction to smart contracts {#introduction-to-smart-contracts}
+Smart contracts turn the terms of an agreement into computer code that executes when the contract terms are met.
```

Comment:
My 2 gwei:

> turn the terms of an agreement into computer code

I don't like the framing here. It makes it sound like smart contract are primarily for legal contracts, which is a misnomer. They're computer programs first & foremost, which are capable of more broader functionality than merely turning terms of an "agreement" into computer code. There doesn't have to be any "agreement" between parties. e.g. is a smart contract wallet "terms of an agreement"? Maybe technically but I don't think it's a straightforward explanation. I feel this definition puts an inaccurate limit on what they're capable of.